modules/covid_research/app/services/covid_research/volunteer/form_crypto_service.rb

```diff
@@ -3,28 +3,26 @@
 module CovidResearch
   module Volunteer
     class FormCryptoService
-      extend AttrEncrypted
-      attr_encrypted :submission, key: Settings.db_encryption_key
+      # attr_encrypted :submission, key: Settings.db_encryption_key
```

Comment:
How does this get migrated?

Reply:
Looking at it again. This "submission" attribute is purely an attribute in Ruby and not in the db?

Reply:
It doesn't necessarily get migrated. This class is used to encrypt job params when they're pushed to redis. We will have to be mindful of any jobs in the queue (which is doesn't seem that this job runs often) during the migration process.

Reply:
So the submission attribute was just an attribute on a PORO purely used to encrypt job params when pushed to the queue in redis